docs/examples/driver_examples/Qcodes example with Stanford SR830.ipynb

```diff
@@ -118,55 +124,99 @@
    "cell_type": "markdown",
    "metadata": {},
    "source": [
-    "## Buffered Acquisition\n",
-    "\n",
+    "### Preparing for reading the buffer and measurement"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
     "The SR830 has two internal data buffers corresponding to the displays of channel 1 and channel 2.\n",
     "\n",
     "Here we present a simple way to measure one of these buffers.\n",
     "\n",
-    "Each buffer can hold 16383 points. The buffers are filled simultaneously. The QCoDeS driver always polls the **entire** buffer, so make sure to reset (clear) the buffer of old data before starting and acquisition."
+    "Each buffer can hold 16383 points. The buffers are filled simultaneously. The QCoDeS driver always pulls the entire buffer, so make sure to reset (clear) the buffer of old data before starting and acquisition.\n",
+    "\n",
+    "We setup channel 1:"
    ]
   },
   {
    "cell_type": "code",
    "execution_count": 4,
-   "metadata": {
-    "collapsed": true
-   },
+   "metadata": {},
    "outputs": [],
    "source": [
-    "# Set up channel 1 \n",
     "sr.ch1_display('X')\n",
     "sr.ch1_ratio('none')"
    ]
   },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "The driver can change the sensitivity automatically according to the R value of the lock-in. So instead of manually changing the sensitivity on the front panel, you can simply run this in your data acquisition or Measurement (max_changes is an integer defining how many steps the autorange can change the sensitivity, the default is 1 step):"
+   ]
+  },
   {
    "cell_type": "code",
    "execution_count": 5,
-   "metadata": {
-    "collapsed": true
-   },
+   "metadata": {},
    "outputs": [],
    "source": [
-    "# Set up the buffer\n",
-    "sr.buffer_SR(512)  # Sample rate (Hz)"
+    "sr.autorange(max_changes=2)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "Setting up the buffer:"
    ]
   },
   {
    "cell_type": "code",
    "execution_count": 6,
-   "metadata": {
-    "collapsed": true
-   },
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "sr.buffer_SR(512)  # Sample rate (Hz)"
+   ]
+  },
+  {
```

Comment:
Is it worth it to also show how the buffer can be filled by using an external trigger (I assume that it can) rather than using a time series?

Reply:
Alright, but you mean a software trigger, right?

Reply:
Does it not have a hardware trigger ? Anyway its fine to wait with this. Perhaps just mention that there are other options will be good enough for now

Reply:
There is a trig button in its front panel. Unfortunately, it is not easy to find one and work with it today. So we can mention the trigger in the notebook for now.

Reply:
How about something like this:
Here we fill the buffer for one second as shown below. In a real use, a trigger can be sent to fill the buffer either on the instrument or a software trigger using the driver: